meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-secureboot

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Balena Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# vi: ft=bash
+
+[ -f "/usr/libexec/os-helpers-logging" ] && . /usr/libexec/os-helpers-logging
+
+parse_efivar_val() {
+	od --address-radix=n --format=u1 $1 \
+		| tr -s "' '" \
+		| cut -d ' ' -f6
+}
+
+# return zero if secure boot is enabled
+secureboot_enabled() {
+	[[ "$(parse_efivar_val /sys/firmware/efi/efivars/SecureBoot-*)" -eq 1 ]]
+	return
```

Comment:
This in an include script and it needs to cater for being called from sh as well as bash - let's make it POSIX compliant and drop bashisms like [[.